modules/View.js

```diff
@@ -116,12 +131,6 @@
      * See Torso.templateRenderer#render for params
      */
     templateRender: function(el, template, context, opts) {
-      // Detach just this view's child views for a more effective hotswap.
-      // The child views will be reattached by the render method.
-      this.detachTrackedViews({ shared: false });
-      // Detach just this view's shared views for a more effective hotswap.
-      // The shared views will be reattached by the render method.
-      this.detachTrackedViews({ shared: true });
       templateRenderer.render(el, template, context, opts);
```

Comment:
Two reasons I want to keep it:
1. backwards compatibility. not such a huge reason but still small.
2. maintaining a separate interface. templateRenderer is actually a different module - one that we have a high likelihood of replacing in the future (with something like the new handlebars library, Bars). The implementation of our templateRender may change but our method hopefully will not.